example/ConfidenceBand.js

```diff
@@ -0,0 +1,11 @@
+// module ConfidenceBand
+
+exports.toolTipFomatter = function(fn) {
```

Comment:
This one is a bit tricky. In fact we can provide more precise type here and even use curried function. But I suppose that it's up to you to change this or not.